src/js/classes/DateTimeValidator.ts

```diff
@@ -159,31 +183,37 @@ class DateTimeValidator extends BaseValidator<Date | string> {
      * @return {this}
      */
     public lessOrEqualTo(name: string, msg?: string): this {
-        const errorMsg = msg ?? 'This value must be less than or equal to the value of ' + name;
-
         this.validateFuncs.push(
             (value, otherValues) => {
                 const otherVal = new Date(getFieldValueFromKeyString(name, otherValues) as Date | number);
-                return {success: isInvalidDate(otherVal) || value <= otherVal, errorMsg};
-            }
+                return {
+                    success: isInvalidDate(otherVal) || value <= otherVal,
+                    errorMsg: msg,
+                    msgName: 'less_or_equal_to_date',
+                    translationContext: {name: name},
+                };
+            },
         );
 
         return this;
     }
 
     /**
      * Check if the fields value is greater than the value of one of its siblings
-     * @param {String|Number} key
+     * @param {String|Number} name
      * @param {String} [msg]
      * @return {this}
      */
-    public greaterThanSibling(key: string | number, msg?: string): this {
-        const errorMsg = msg ?? 'This value must be greater than the value of its sibling ' + key;
-
+    public greaterThanSibling(name: string | number, msg?: string): this {
```

Comment:
why renaming `key` to `name`? I feel like it's more intuitive that a key can be a string or a number, if it's called name I wouldn't expect it to be able to be a number as well.